src/analyzer/analyzerrhythmbpm.cpp

```diff
@@ -12,26 +13,47 @@
 namespace {
 constexpr bool sDebug = false;
 constexpr int kHistogramDecimalPlaces = 2;
-constexpr double kCorrectBeatLocalBpmEpsilon = 0.05; //0.2;
 const double kHistogramDecimalScale = pow(10.0, kHistogramDecimalPlaces);
+// We look for the first beat whitin this to be the first beat
+constexpr double kCorrectBeatLocalBpmEpsilon = 0.05; //0.2;
```

Comment:
What is this commented out 0.2?

Reply:
Artifact from beatutils, haven't personally experimented with this value, remove or leave it for later reference?

Reply:
Remove it or explain it, but do not leave random numbers in the code.